shared/video-sdk/develop/stream-raw-audio-and-video/project-implementation/windows.mdx

```diff
@@ -301,15 +318,19 @@ This sections describes the steps required to use the relevant libraries and dec
         }
         ```
 
-    1. To register the observers at startup, in `AgoraImplementationDlg.cpp`, add the following at the of `setupVideoSDKEngine()`:
+    1. To register the observers at startup, in `AgoraImplementationDlg.cpp`, add the following at the start of
```

Comment:
This is incorrect as observers cannot be registered before engine initialization. In the SetupVideoSDKEngine method, the observer should be registered after engine initialization, but it is currently being registered before initialization.

Reply:
Actually wrong at the very first removal of original place. I ket this call in the `JOIN` button listerner event. We can not keep it in   ``setupVideoSDKEngine()` ot at the startu[ (OnInitDialog ()). Becaause a user can join or leave the channel multiple time but initDlg will be called once .Suppose  a user  left the channel and re-join again... then if it kept in start up , it will not enabled . For the same reason we should keep the disabling the observers   `EnableAudioVideoCapture(FALSE)' only in the LEAVE button listen event. I had kept them at correct place. Please don't make these changes.